### DISCUSSION TITLE
arakasi72's rtinst script not working

### DISCUSSION THREAD

**Discussion by tonton63000** [Novik/ruTorrent#2482] [Q&A] [answered]
Hello, sorry if I'm in the wrong place. I'm not an expert and so I've been using arakasi72's script from the start and never had a problem. the new script does not work at all.
Tried on Ubuntu 18, 20 and 22
I only have errors
18.04.2023 12:54:59] WebUI started.
[18.04.2023 12:54:59] throttle: The plugin cannot start for an unknown reason.
[18.04.2023 12:54:59] autodl-irssi: __init2: ex: Exception info: No content elem with id autodl-prefs-contents-upload
[18.04.2023 12:56:17] Bad response from server: (500 [error,list]) Link to XMLRPC failed. Maybe, rTorrent is down?

Being a newbie if anyone had a solution.

THANKS

**Comment by stickz** [comment, marked as answer]
This script doesn't work properly with the latest version of ruTorrent. 

I would recommend using swizzin instead. They have a community support channel on Discord, if you have any questions.
https://github.com/swizzin/swizzin

Anther possible alternative is docker-rutorrent. This option is less feasible. But some people don't want to modify the operating system layer.
https://github.com/crazy-max/docker-rtorrent-rutorrent

I've contributed to both projects, to ensure they work properly with the latest version of ruTorrent.

**Comment by tonton63000** [comment]
Thank you for your prompt response . I will install 22.04 and the swizzin script
I will keep you informed . Good day

**Comment by tonton63000** [comment]
Thank you very much. After several tries on 22.04 impossible to make swizzin work
Probably a mistake on my part
Install on 20.04 Works great
Just a small screenshot error, probably because I didn't install ffmpeg
But works for me .Thanks for the help and work

**Comment by tonton63000** [comment]
Just a small screenshot error, probably because I didn't install ffmpeg
error solved by running the script and installing FFmpeg

**Comment by Suitear** [comment]
Thanks, guys. I stick to this rtinst script because I used to the rtorrent location (home/user/rtorrent/download) and don't know if I change it to other scripts, how can I modify the new script to adapt to my existing 12000+ torrents.

**Comment by jaum20** [comment]
Solution fot the throttle plugin problem: https://github.com/arakasi72/rtinst/issues/614#issuecomment-1412337985
